File: Leetcode/1.two-sum.py
#
# @lc app=leetcode id=1 lang=python
#
# [1] Two Sum
#

# @lc code=start
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        result = []
        n = len(nums)

        for i in range(n):
            for j in range(i+1,n):
                sum = nums[i] + nums[j]
                if sum==target:
                    a=i
                    b=j
        result.append(a)
        result.append(b)
        
        return result
        
# @lc code=end

File: Leetcode/1.two-sum.java
/*
 * @lc app=leetcode id=1 lang=java
 *
 * [1] Two Sum
 */

// @lc code=start
class Solution {
    public int[] twoSum(int[] nums, int target) {
        for (int i = 0; i < nums.length; i++) {
            for (int j = i; j < nums.length; j++) {
                if (nums[i]==nums[j]) {
                    occ++;
                }
            }
        }
        int[] res = {};
        return res;
    }
}
// @lc code=end

